docs/release_notes.rst

```diff
@@ -170,6 +170,14 @@ Data Cleaning
 * The :ref:`boiler_fuel_eia923` table now includes the ``prime_mover_code`` column. This
   column was previously incorrectly being associated with boilers in the
   :ref:`boilers_entity_eia` table. See issue :issue:`2349` & PR :pr:`2362`.
+* Fixed column naming issues in the :ref:`electric_operating_revenues_ferc1` table.
+* Made minor calculation fixes in the metadata for :ref:`income_statement_ferc1`,
+  :ref:`utility_plant_summary_ferc1`, :ref:`electric_operating_revenues_ferc1`,
+  :ref:`balance_sheet_assets_ferc1`, :ref:`balance_sheet_liabilities_ferc1`, and
+  :ref:`electric_operating_expenses_ferc1`.
```

Comment:
@e-belfer if you have them handy, can you add some PR or issue refs to these changes so we / others can see what code changes correspond to these updates?

Reply:
Oops, done!

Reply:
Thanks!